spring-integration-core/src/test/java/org/springframework/integration/channel/config/ChannelParserTests.java

```diff
@@ -62,65 +68,76 @@
  *
  * @see ChannelWithCustomQueueParserTests
  */
+@RunWith(SpringRunner.class)
```

Comment:
`SpringRunner` is from SF-4.3. Doesn't fit cherry-picking into `4.2` and `4.1` :smile: 
Will fix on merge together with update Copyrights.